Reflections/week_1_reflection.md

```diff
@@ -0,0 +1,27 @@
+## What have you accomplished this week? 
+
+* I got in user input and turned it into a list 
+* I iterated over the dataframe to pull information from a certain column (the recipes)
+* I've been researching web crawlers and new ways to scrape information, and how the process works 
+
+## What challenges or difficulties did you face? 
+
+Right now I'm having issues actually manipulating the dataframe, and comparing the ingredients data. The 
+reason for this is because it contains words like 'cups', 'tsp', etc. I'll have to consider stripping those 
+in the comparison. I didn't get it fixed this week, though I have reached out with a few students and plan 
+on going to TAs. 
+
+## What do you plan on accomplishing in the following week? 
+
+* Figuring out how to compare user's list against the ingredients list, even if it still has the 
+caveat of ingredient amount. 
+  
+* Calculating the percentage matched. 
+
+* Adding more data to compare against. 
+

```

Comment:
These three goals look attainable but some other components of "S.M.A.R.T." goals are missing. For the next reflection, do not forget to add the with estimated number of hours/day to complete each goal.